android-sdk/src/main/java/com/checkout/android_sdk/Utils/CheckoutTheme.kt

```diff
@@ -0,0 +1,65 @@
+package com.checkout.android_sdk.Utils
+
+import android.content.Context
+import android.content.res.Resources
+import android.util.TypedValue
+import com.checkout.android_sdk.R
+
+internal class CheckoutTheme(val context: Context) : CheckoutThemeInterface {
```

Comment:
Is context field used outside of the class?
It would be nice to make it private if possible.